packages/atomic/src/components/facets/atomic-segmented-facet/atomic-segmented-facet.pcss

```diff
@@ -0,0 +1,35 @@
+@import '../../../global/global.pcss';
+@import '../facet-common.pcss';
+
+.box-container {
+  @apply flex items-stretch;
+}
+
+.box-container li:first-child .value-box:first-child {
+  border-radius: 3px 0px 0px 3px;
+}
+
+.box-container li:last-child .value-box:last-child {
+  border-radius: 0px 3px 3px 0px;
+}
+
+.value-box .value-label,
+.value-box-count {
+  @apply w-full;
+}
+
+.value-box {
+  @apply hover:border-blue-400 rounded-none;
```

Comment:
has to be an atomic theme color https://github.com/coveo/ui-kit/blob/master/packages/atomic/tailwind.config.js#L7